ipatests/test_integration/test_bugzilla.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2015  FreeIPA Contributors see COPYING for license
+#
+
+import time
+from tempfile import NamedTemporaryFile
+
+from ipatests.pytest_plugins.integration import tasks
+from ipatests.pytest_plugins.integration.env_config import get_global_config
+from ipatests.test_integration.base import IntegrationTest
+
+config = get_global_config()
+
+
+class LayoutsBaseTest(IntegrationTest):
+    @classmethod
+    def install(cls, mh):
+        # tests use custom installation
+        pass
+
+    def replication_is_working(self):
+        test_user = 'replication-testuser'
+        self.master.run_command(
+            ['ipa', 'user-add', test_user, '--first', 'test', '--last', 'user']
+        )
+
+        time.sleep(60)  # make sure the replication of user is done
```

Comment:
Maybe Ok for now, but doesn't look very robust (it doesn't ensure anything, only hopes that 60s is enough).